api-admin/src/main/java/bio/terra/pearl/api/admin/controller/forms/ConfiguredSurveyController.java

```diff
@@ -30,6 +31,27 @@ public ConfiguredSurveyController(
     this.surveyExtService = surveyExtService;
   }
 
+  /**
+   * gets all StudyEnvironmentSurveys for the given study, along with their associated Survey (minus
+   * content)
+   */
+  @Override
+  public ResponseEntity<Object> findWithNoContent(
```

Comment:
Unsure how I feel about the name of this function - feels a bit ambiguous, but otherwise looks good

Reply:
concur, I'm not a fan, but can't think of anything better